src/main/java/frc/robot/subsystems/ArmRotate.java

```diff
@@ -0,0 +1,79 @@
+package frc.robot.subsystems;
+
+import java.util.function.DoubleSupplier;
+
+import org.littletonrobotics.junction.Logger;
+
+import com.chopshop166.chopshoplib.commands.SmartSubsystemBase;
+import com.chopshop166.chopshoplib.motors.CSSparkMax;
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+import com.chopshop166.chopshoplib.sensors.IEncoder;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.maps.subsystems.ArmRotateMap;
+import frc.robot.maps.subsystems.ArmRotateMap.Data;
+
+public class ArmRotate extends SmartSubsystemBase {
+
+    private SmartMotorController armMotor = new CSSparkMax(1, MotorType.kBrushless);
+    private IEncoder armAngle;
+    private ArmRotateMap map;
+    private PIDController pid = new PIDController(0, 0, 0);
+    final double MOVE_SPEED = 0.5;
+    final double COMPARE_ANGLE = 5;
+    final double TOP_ANGLE = 85;
+    final double BOTTOM_ANGLE = 5;
```

Comment:
These should both be defined in the map as well. They do not need to be part of the data layer since they won't change.